Loop/Views/SupportScreenView.swift

```diff
@@ -33,10 +39,33 @@ struct SupportScreenView: View, HorizontalSizeClassOverride {
         .navigationBarTitle(Text("Support", comment: "Support screen title"))
         .environment(\.horizontalSizeClass, horizontalOverride)
     }
+    
+    private var adverseEventReport: some View {
+        Button(action: {
+            guard let url = self.adverseEventReportViewModel.reportURL else {
```

Comment:
[nit] It's nice that you did this, but wouldn't it be a code configuration (compile-time) error if `reportURL` is `nil`?  Seems unlikely this URL would be invalid...

Reply:
That or something slips through by mistake. I guess I was thinking what would be worse, the user tapping the cell and nothing happening or it presenting an alert. I chose the later.